amqp.go

```diff
@@ -106,7 +107,8 @@ func (amqp *AMQP) Publish(options PublishOptions) error {
 		publishing.DeliveryMode = amqpDriver.Persistent
 	}
 
-	return ch.Publish(
+	return ch.PublishWithContext(
```

Comment:
Changed to new method given deprecation of the original `Publish` method. See [docs](https://pkg.go.dev/github.com/rabbitmq/amqp091-go@v1.5.0#Channel.Publish).